deepspeed/runtime/progressive_layer_drop.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+from deepspeed.runtime.constants import PLD_THETA, PLD_GAMMA
+from deepspeed.utils import log_dist
+
+
+class ProgressiveLayerDrop(object):
+    def __init__(self, pld_params):
+        self.theta = pld_params[PLD_THETA]

```

Comment:
Can you add a `super().__init__()` to the beginning of this initialization? It aids in later development if someone wants to inject a class hierarchy.